frontendservice/public/index.html

```diff
@@ -11,24 +11,40 @@
 
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+  <head>
+    <style>
+      body {
+        background-image: url("https://img.freepik.com/free-photo/black-friday-composition-with-four-bags-cart_23-2147709334.jpg?size=626&ext=jpg");
+        background-repeat: no-repeat;
+        background-size: cover;
+      }
+    </style>
```

Comment:
maybe include the background styling in App.vue instead?

Reply:
or can create a static stylesheet and then include it in the index.html file

Reply:
or can create a static folder, create a style sheet for background image in that folder and include the style sheet in index.html instead?

Reply:
Done Bavanya